xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -63,6 +63,8 @@ class CSinkControlProtocol : public Protocol
     VOLUME,
     FLUSH,
     TIMEOUT,
+    SILENCECOUNTERUPDATED,
+    NOISECHANGED,
```

Comment:
here "changed", the other "updated". the the goal to have as many different names as possible? :)
I would call the signal SetXXX, that also fits initial setting of those parameters